aggregator/src/datastore.rs

```diff
@@ -293,6 +293,12 @@ pub struct Transaction<'a, C: Clock> {
 }
 
 impl<C: Clock> Transaction<'_, C> {
+    /// Execute a sequence of SQL statements.
+    #[tracing::instrument(skip(self), err)]
+    pub async fn batch_execute(&self, query: &str) -> Result<(), Error> {
```

Comment:
This is a very non-portable operation; I think we should not include it in the `Datastore` struct. (the purpose of the datastore layer is to abstract the operations from the raw SQL that implements them, to allow other storage backends to more easily be written)